### PR TITLE
Create website fetching data from Poke API

### PR DESCRIPTION
**Pull request by Javivy**
##In this project my partner and I did:

  - [x] Set up the project with webpack, jest, and linters
  - [x] We created the Navbar and the Footer of the home page
  - [x] We created the basic UI of the website
  - [x] @Javivy Fetched the data from the API and displayed it on the home page
  - [x] @MAHDIA001 Created the Popup for the comments
  - [x] @Javivy Fetched the likes from the **Involvement API**
  - [x] @MAHDIA001 Fetched the comments from the **Involvement API**
  - [x] @Javivy Created the counter for the items on the home page
  - [x] @MAHDIA001 Created the counter for the comments in the popup 

[Video Presenting the project](https://drive.google.com/file/d/1dUKuDBK0ZP76KQ3x6PV6or6SjNKhVpco/view?usp=sharing)

We successfully worked as a team, communicating and getting agreement on what to do on the project, understanding, and learning throughout the process.

### CODE REVIEW

File: src/tests/counterComment.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+describe('count the number of comments', () => {
+  test('It should fetch 2 comments', () => {
+    getComments(2).then((res) => {
+      expect(res.length).toBe(2);
+    });
+  });
+  test('return number of comments', () => {
+    expect(cmtCount()).toBe(2);
+  });
```
Comment (jerryowusu) [inline src/tests/counterComment.test.js:30]: Kindly ensure that you write tests for all edge cases. You should at least consider about the scenario in which there are 0 items on the page (counter function should return 0). This applies to both card counter and comments counter.

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @Javivy and @MAHDIA001 :wave: 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

soclose


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @Javivy , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
